### PR TITLE
add autogenerated (by AI) install.md with easy to follow instructions

### PR DESCRIPTION
**Pull request by umputun**
This is a prototype for an easy installation guide. I'm not sure how easy it is in real life and if any steps are missing.

Please note - the goal of this document is to help people who are not very familiar with Docker to get started.

### CODE REVIEW

**Comment by umputun** [comment]
A typical message I get from such users that triggered this PR is something like this: "Can you suggest what can be looked at/read to understand the installation and use of tg-spam? Go and Docker are still unfamiliar to me, but I would like to run this bot in the chat. Thank you in advance."

File: site/docs/index.md [hunk 1]
```diff
@@ -280,7 +280,6 @@ meta:
 
 openai:
       --openai.token=               openai token, disabled if not set [$OPENAI_TOKEN]
-      --openai.apibase=             custom openai API base, default is https://api.openai.com/v1 [$OPENAI_API_BASE]
```
Comment (paskal) [inline site/docs/index.md:283]: I think this line shouldn't be removed.

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+5. Restart your computer when asked
+
+### For Mac:
+1. Go to [Docker Desktop for Mac](https://www.docker.com/products/docker-desktop/)
+2. Click the "Download for Mac" button
+3. Once downloaded, drag Docker to your Applications folder
+4. Double-click Docker in Applications to start it
+5. Follow any prompts that appear
+
+### For Linux:
```
Comment (paskal) [inline INSTALL.md:43]: Copypasting Ubuntu instructions is a bad idea, I think it will lead to discrepancies and something breaking long-term. Your call.

**Review by paskal — APPROVED** [review]
Overall, good. I Think we might want to translate it to Russian and other popular telegram languages, e.g. Russian, Hindi and Indonesian.

**Pull request merged**